build-and-run

```diff
@@ -5,9 +5,18 @@ then
   git checkout "$1"
 fi
 
+# Exit on errors
+set -e
+
+# Print the commands as they are executed
+set -x
+
 echo "Building frontend..."
 
-npm run build --prefix frontend
+pushd "frontend" > /dev/null 2>&1
+npm install
```

Comment:
It might be better to use `npm ci` instead, so it really installs only what's in the `package-lock.json`

Reply:
That does significantly increase startup time, but I think if it's used in situations like today, that's a fine tradeoff :+1:

Reply:
Fixed in [71a1b39](https://github.com/kiesraad/abacus/pull/492/commits/71a1b39b8c5e5e35073cfb902f2744809efd0517)